war.rb

```diff
@@ -74,4 +74,4 @@
   puts "-------------------------"
   puts "----Player 2 WINS!!!!----"
   puts "-------------------------"
-end
\ No newline at end of file
+end
```

Comment:
Was this not there already?

Reply:
The Readme was there. I put 
"$ ruby war.rb"
In markdown, with syntax, ie
```
$ruby war.rb
```

Reply:
The PR shows this end was added, but I had it there already. I might just be looking at it wrong.